From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Manish Bhardwaj <mbhardwaj@nvidia.com>
Date: Thu, 22 Jan 2026 03:13:01 +0000
Subject: [PATCH 196/254] NVIDIA: SAUCE: firmware: bpmp: include tegra_hv_ivc.h
 header file

new tegra_hv_ivc.h header file has been added due to
tegra_hv.c driver refactoring. so use tegra_hv_ivc.h
file instead of older hv-ivc.h file.

Jira ESLC-8959

Change-Id: If468f41784e99a709bb060c23ce402d5d6efe20f
Signed-off-by: Manish Bhardwaj <mbhardwaj@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/3rdparty/canonical/linux-noble/+/3533374
Reviewed-by: Sandeep Trasi <strasi@nvidia.com>
Reviewed-by: svcacv <svcacv@nvidia.com>
GVS: buildbot_gerritrpt <buildbot_gerritrpt@nvidia.com>
Reviewed-by: Suresh Venkatachalam <skathirampat@nvidia.com>
(cherry picked from commit 9ceec99a96fe3d0f25f5758ba4c527ddbe0ea5b0)

diff --git a/drivers/firmware/tegra/bpmp-tegra194-hv.c b/drivers/firmware/tegra/bpmp-tegra194-hv.c
index d171aa532e2e8..47312cf738010 100644
--- a/drivers/firmware/tegra/bpmp-tegra194-hv.c
+++ b/drivers/firmware/tegra/bpmp-tegra194-hv.c
@@ -1,5 +1,5 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
-/* SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved. */
+/* SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved. */
 
 #include <linux/interrupt.h>
 #include <linux/of_address.h>
@@ -7,7 +7,7 @@
 
 #include <soc/tegra/bpmp.h>
 #include <soc/tegra/ivc.h>
-#include <soc/tegra/virt/hv-ivc.h>
+#include <soc/tegra/virt/tegra_hv_ivc.h>
 #include "bpmp-private.h"
 
 /* utilizing the struct tegra_ivc *ivc in struct tegra_bpmp_channel
-- 
2.43.0

